src/idlcxx/src/streamers.c

```diff
@@ -263,14 +274,14 @@ write_basic_type_streaming_functions(
   const char* read_accessor,
   instance_location_t loc)
 {
-  const char* fmt = "  ::org::eclipse::cyclonedds::core::cdr::%2$s(streamer, %1$s);\n";
+  const char* fmt = "  %2$s(streamer, %1$s);\n";
   const char* read_fmt = fmt;
   if ((idl_mask(type_spec) & IDL_BOOL) == IDL_BOOL &&
       loc.type & SEQUENCE) {
     read_fmt =
       "  {\n"
       "    bool b;\n"
```

Comment:
Since `b` is not initialized here, but it is assigned to another variable in the lines below. The compiler would warn with  "may be used uninitialized in this function" (though b gets initialized in the streamer function in the below line. Can we default initialize this bool value?

Reply:
Sure, I will add initialization of the variable on creation.